docs/commands.rst

```diff
@@ -376,9 +368,9 @@ Examples:
 
 ::
 
-   set unif_rule Bool ≡ T $t ↪ $t ≡ bool
-   set unif_rule $x + $y ≡ 0 ↪ $x ≡ 0; $y ≡ 0
-   set unif_rule $a → $b ≡ T $c ↪ $a ≡ T $a'; $b ≡ T $b'; $c ≡ arrow $a' $b'
+   set unif_rule Bool ≡ T $t ↪ begin $t ≡ bool end
+   set unif_rule $x + $y ≡ 0 ↪ begin $x ≡ 0; $y ≡ 0 end
+   set unif_rule $a → $b ≡ T $c ↪ begin $a ≡ T $a'; $b ≡ T $b'; $c ≡ arrow $a' $b' end
```

Comment:
semi-colons missing at the end

Reply:
Here again we have an `end`, so there is no need for a semicolon